notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -123,55 +123,6 @@
         "to generate a cost estimate based on your projected usage."
```

Comment:
<p>I will add this notebook tomorrow, I won't merge this PR until that is in.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>